Examples/test-suite/fragments.i

```diff
@@ -34,3 +34,86 @@ int foo(int hola)
 }
 
 %}
+
+/* Instantiate multiple fragments at once using multiple keywords */
+typedef int int_infrag3;
+typedef int explicit_frag3;
+
+%fragment("explicit_frag1","header", noblock=1) {
+typedef int explicit_frag1;
+}
+
+%fragment("explicit_frag2","header", noblock=1, noblock=1) {
```

Comment:
Is the double `noblock=1` intentional?

Reply:
It's been a long time since I wrote this test, but perhaps it's to test that nothing breaks with duplicate non-fragment kwargs? I can delete the duplication.

Reply:
I remove the additional noblock in d59cbf1c39efd4541275b9868514dd24b9880ba7